DESCRIPTION

```diff
@@ -19,7 +24,13 @@ Encoding: UTF-8
 LazyData: true
 biocViews: Software, Infrastructure
 Imports:
-    Biostrings
+    Biostrings,
+    BiocManager,
+    biomaRt,
+    protr,
+    doParallel,
```

Comment:
Not sure if you actually use doParallel or foreach in your code.

Reply:
You "resolved" this conversation without a comment or response.